.travis.yml

```diff
@@ -1,9 +1,6 @@
 sudo: required
 services:
   - docker
-branches:
-  only:
-    - master
```

Comment:
CI is not triggered on tag push included this configuration. I cannot make a GitHub release.